CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## 1.5.3 2018-12-20
+
+Linkerd 1.5.3 includes a Finagle upgrade that fixes some memory related issues and adds support for 
+more efficient HTTP/1.1 routing for large HTTP requests. This release also improves 
+Linkerd's start up script to can run with newer versions of Java. Finally, this release adds a
+new feature for gRPC response classifiers that allow for added flexibility in classifying user 
+defined gRPC status codes.
+
+Full release notes:
+
+HTTP/1.1
+  * Adds a new config option `streamAfterContentLengthKB` that sets a threshold at which HTTP 
+  messages will be streamed by Netty if exceeded. This avoids having to allocate large HTTP messages
```

Comment:
maybe: "... will be streamed instead of being fully buffered in memory."

Reply:
This allows for the streaming of large messages, even when chunk-encoding is not used.